### PR TITLE
Update changelog for 1.10.4

### PR DESCRIPTION
**Pull request by award999**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+
+### Added
+
+- `Swift` terminal profile
+- Set Swift environment variables in the integrated terminal
+- Show macro expansions
+
+### Fixed
+
+- Parameterized tests inherit tags
```
Comment (plemarquand) [inline CHANGELOG.md:13]: nit, clarification: Parameterized `swift-testing` tests inherit tags

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 1.10.4 - 2024-07-08
+
+### Added
+
+- `Swift` terminal profile
+- Set Swift environment variables in the integrated terminal
+- Show macro expansions
```
Comment (plemarquand) [inline CHANGELOG.md:9]: nit: Show macro expansions via Quick Action

Comment (plemarquand) [inline CHANGELOG.md:9]: I think we should also add a bit of documentation to [settings.md](https://github.com/swiftlang/vscode-swift/blob/main/docs/settings.md) about how to enable macro expansion w/ the LSP arguments setting:

```
"swift.sourcekit-lsp.serverArguments": [
  "--experimental-feature", "show-macro-expansions"
]
```

Comment (plemarquand) [inline CHANGELOG.md:9]: Also give a shoutout to @lokesh-tr for his great work on this!

**Review by plemarquand — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (adam-fowler) [reply CHANGELOG.md:9]: Can we remove reference to the macro expansion. It is reliant on sourcekit-lsp changes that won't make it into 6.0.

File: CHANGELOG.md (hunk 1 above)
Comment (award999) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (award999) [reply CHANGELOG.md]: Removed

Comment (award999) [reply CHANGELOG.md]: Removed

Comment (adam-fowler) [reply CHANGELOG.md]: I think it is better this is mentioned in any GSoC posts on the forums

**Review by adam-fowler — APPROVED** [review]

**Review by plemarquand — APPROVED** [review]

**Pull request merged**